src/components/lco/PageBanner.tsx

```diff
@@ -36,62 +44,93 @@ type PageBannerProps = Pick<ClimbType, 'ancestors'>
  */
 export const PageBanner: React.FC<PageBannerProps> = ({ ancestors }) => {
   const orgs = findLCOs(LCO_LIST, ancestors)
-  if (orgs.length === 0) return null
   return (
-    <div className='mt-8 bg-area-cue/20 p-6 rounded-box'>
-      <div className='mt-2 mb-1 text-xs text-base-300'>Local climbing organizations</div>
-      <div className='flex flex-col'>
-        {orgs.map(orgProfile => <IndividualBanner key={orgProfile.id} profile={orgProfile} />)}
+    <div className='grid pt-6 pb-4 lg:pb-16 lg:pt-16'>
+      <div className='col-span-full flex justify-start items-center pb-6'>
+        <h3 className='mr-4'>Local climbing organizations</h3>
+        <InformationCircleIcon className='h-6 w-6' />
```

Comment:
Why don't we just add a tool tip and link to
"Learn more about our [initiative](https://openbeta.substack.com/p/openbeta-and-lcos)."

### Tooltip usage: 

- Tooltip with a simple string content: https://github.com/OpenBeta/open-tacos/blob/7db52ce0c7883c79cb4ee9b365c9de784da4aa88/src/components/edit/form/AreaDesignationRadioGroup.tsx#L36
- Tooltip with JSX content: https://github.com/OpenBeta/open-tacos/blob/develop/src/components/edit/form/LegacyFAInput.tsx